app/src/test/kotlin/edu/stanford/bdh/engagehf/health/HealthUiStateMapperTest.kt

```diff
@@ -65,12 +65,12 @@ class HealthUiStateMapperTest {
         // Given
         val records = listOf(
             createWeightRecord(
-                year = 2024,
-                day = 1,
+                year = ZonedDateTime.now().year,
+                day = ZonedDateTime.now().dayOfMonth,
```

Comment:
I also noticed these tests starting to fail now in september 😅 Would it make sense to only create the zoned date time once and reference then throughout the file? Did something similar [here](https://github.com/StanfordSpezi/SpeziKt/pull/94/files#diff-53b25f9e313f5c831ce11cac1845ef11a4769c7e545440e42fbccbc02001cab3R20)

Reply:
@eldcn  Yes, makes completly sense.  my suggestion would be to simply replace your changes with mine when merging.  🚀